packages/helpers/src/index.ts

```diff
@@ -28,6 +28,31 @@ export interface Options {
 
 const BECH32_LIMIT = 1023;
 
+export function minimalScriptCapacity(
+  script: Script,
+  { validate = true }: { validate?: boolean } = {}
+): bigint {
```

Comment:
Using bigint is a legacy issue, @homura maybe we should change all bigint to BI? so that we don't need to add xxxCapatible in the future.

Reply:
we'll keep the API same as the other functions for now, so we won't change it for now, let's keep `minimalScriptCapacity` + `minimalScriptCapacityCompatible`